### PR TITLE
Remove oga dependency

### PR DESCRIPTION
**Pull request by madwork**
Ruby comes with REXML as a default XML parser, it’s much slower than oga or nokogiri but for a small library, with a non intensive use of XML parsing (most of the time the result will be stored) it should be quite enough, and it removes any external dependency.

Adding such dependency into a small (but quite handy 😄) library is sometime a big issue, most certainly with dependency like nokogiri or oga which could brings some heavy dependencies. And with projects (rails per example) which already include nokogiri, it starts to be a dependency nightmare only to parse an XML file once in a day.

### CODE REVIEW

**Comment by hakanensari** [comment]
:+1:

I'm assuming you tried this with the biggest dataset and didn't choke.

**Comment by madwork** [comment]
It's a bit long on the whole dataset, but it works great! 👍

File: lib/fixer/feed.rb [hunk 1]
```diff
@@ -16,9 +16,9 @@ def initialize(type = :current)
     end
 
     def each
-      document.xpath('/Envelope/Cube/Cube').each do |day|
+      REXML::XPath.each(document, '//Cube/Cube[@time]') do |day|
```
Comment (dentarg) [inline lib/fixer/feed.rb]: why the change in xpath expression? isn't doing `//` kinda slow?

Comment (madwork) [inline lib/fixer/feed.rb]: Yes, I'm surprise about it, but you are right, it's slower with this path, it's faster with this path `/gesmes:Envelope/Cube/Cube[@time]`.

Comment (hakanensari) [inline lib/fixer/feed.rb]: I reverted to full xpath
